apps/etterlatte-behandling/src/main/kotlin/behandling/omberegning/OmberegningRoutes.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.etterlatte.behandling.omberegning
+
+import io.ktor.server.application.call
+import io.ktor.server.request.receive
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.post
+import io.ktor.server.routing.route
+import no.nav.etterlatte.behandling.GenerellBehandlingService
+import no.nav.etterlatte.libs.common.behandling.Omberegningshendelse
+
+fun Route.omberegningRoutes(
+    behandlingService: GenerellBehandlingService,
+    omberegningService: OmberegningService
+) {
+    route("/omberegning") {
+        post {
+            val request = call.receive<Omberegningshendelse>()
+            val forrigeBehandling = behandlingService.hentBehandlingerISak(request.sakId)
+                .maxByOrNull { it.behandlingOpprettet }!!
```

Comment:
Tror ikke Sebby sin keyresult liker `!!` i routen her. Sjekke for null og gi en tydelig feilmelding i steden?